lib/cdc/util/parse_test.go

```diff
@@ -123,6 +123,22 @@ func TestParseField(t *testing.T) {
 			},
 			expectedValue: `{"type":"Feature","geometry":{"type":"Point","coordinates":[123,-39]},"properties":null}`,
 		},
+		{
+			name: "json",
+			field: debezium.Field{
+				DebeziumType: string(debezium.JSON),
+			},
+			value:         `{"foo": "bar", "foo": "bar"}`,
+			expectedValue: `{"foo":"bar"}`,
+		},
+		{
```

Comment:
IIRC the issue was that we weren't handling arrays of objects or arrays of arrays, can we add a test for those?